install_config/install/advanced_install.adoc

```diff
@@ -681,6 +681,154 @@ openshift_examples_modify_imagestreams=true
 |===
 
 
+[[advanced-install-containerized-glusterfs-persistent-storage]]
+=== Configuring Containerized GlusterFS Persistent Storage
+
+GlusterFS can be configured and installed in the {product-title} cluster for
+xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[peristent storage] and
+xref:../storage_examples/gluster_dynamic_example.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamic provisioning].
+ifdef::openshift-enterprise[]
+This option utilizes
+link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.1/html/container-native_storage_for_openshift_container_platform_3.4/[Red Hat Container Native Storage (CNS)] to create containerized GlusterFS storage
+pods that back the registry.
+endif::[]
+ifdef::openshift-origin[]
+See link:https://github.com/gluster/gluster-kubernetes[Running Containerized GlusterFS in Kubernetes] for additional information on containerized storage
+using GlusterFS.
+endif::[]
+
+After successful installation, all required components to start using and
+provisioning storage will exist, including `Endpoints`, `Services` and a
+`StorageClass`.
```

Comment:
Omg that poor missing Oxford comma. :shame:

Updated! And I think I'll leave it in since it mentions some of the objects that will be created, but I don't feel super strongly.